### DISCUSSION TITLE
Meeting: October 29, 2024

### DISCUSSION THREAD

**Discussion by earth2marsh** [OAI/sig-moonwalk#159] [Announcements]
## Weekly meetings happen on Tuesdays

**Time:** 5pm GMT / 9am Pacific
**Zoom:** [NB: this meeting info changed to the LFX system on Oct 22, 2024] https://zoom-lfx.platform.linuxfoundation.org/meeting/98869769538?password=d769f940-7da7-4de8-9fab-685a1cc8f33e

### Accessibility & Etiquette

* Participants must abide by our [Code-of-Conduct](https://github.com/OAI/OpenAPI-Specification/blob/main/CODE_OF_CONDUCT.md#code-of-conduct).
* Meetings are recorded for future reference, and for those who are not able to attend in-person.
* We invited challenges to language or behaviour that is harmful or not inclusive during this meeting.
* We look forward to your participation, but please consider these acts of etiquette:
  * Remain on mute when not speaking to prevent interruptions.
  * Blur your background to reduce visual distractions.
  * Use the Zoom meeting "Raise Hand" feature to notify the group when you wish to speak.

| Blur My Background | Raise Hand |
|-|-|
| <img width="323" alt="Screenshot of Zoom UI showing the 'Stop Video' and 'Blur My Background' control" src="https://github.com/OAI/OpenAPI-Specification/assets/7367/7e43dbbb-6529-46e6-8b04-4c1aa852d9dd"> | <img width="323" alt="Screenshot of Zoom UI showing the 'Reaction' and 'Raise Hand' control" src="https://github.com/OAI/OpenAPI-Specification/assets/7367/f991722f-4651-40aa-9bc4-7e9a2a165a6a"> |

### Agenda Structure

| Topic | Owner | Decision/NextStep |
|-------|---------|---------|  
| | | |
Intros and governance meta-topics (5 mins) | SIG members | -- |
General discussion topics (list below) | SIG members | -- |
Wrap-up and seed the next agenda (5 mins) | SIG members | -- |

Whether attending or not, **anyone can comment on this issue prior to the meeting to suggest topics or to add comments on planned topics or proposals**.

**Comment by darrelmiller** [comment]
Strawperson IDL for Document Interface and ADA Interface https://github.com/darrelmiller/oasbridge/blob/main/OpenAPIInterrface.idl

**Comment by earth2marsh** [comment]
[Video](https://zoom.us/rec/share/RpaK6JpJRUxKb9UjGKSywznEw_WMoTO3D6q-85orLtSo8dBP3wvtzl400Cp8YylP.z9WvohYBIG9jd_mg) is now available.
